dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.idl

```diff
@@ -118,6 +118,24 @@ namespace Microsoft.Windows.AppNotifications.Builder
         AppNotificationComboBox SetSelectedItem(String id);
     };
 
+    [contract(AppNotificationBuilderContract, 2)]
+    runtimeclass AppNotificationComboBox2
```

Comment:
AppNotificationComboBox2 isn't the greatest name. I'm open to suggestion, if someone has a better idea. The main difference is that this version keeps the items in the order they were added by the caller, rather than re-ordering them arbitrarily.

Reply:
My thoughts were that this is a bug to be fixed and didn't need an entire other object to be designed. Anyways, if we want to follow std library style we can do AppNotificationUnorderedComboBox. That would imply that the AppNotificationComboBox has some ordering (which it does) and make sense as a separate class.

But I hope that we don't have to do this, I'd rather just fix the impl inside the original ComboBox.

Reply:
AppNotificationUnorderedComboBox, unordered implies that the developers doesn't have control on the order. Usually, it means ordered by hashtags for efficiency. In this case we very much want the developers to control the order.

Reply:
You can't keep the original runtimeclass because we can't expose the internal map anymore as the innards of the class have changed.

If we want to keep exposing a properties for our C# friends, then it would have to be a vector (of KeyValuePairs) instead of a map.

Either way, that's a foundamental change to the API and can only be achieved with a new runtimeclass

Reply:
Resolving, there's noneed for a new runtimeclass.